app/pages/SavedProgressPage.scala

```diff
@@ -0,0 +1,32 @@
+/*
+ * Copyright 2023 HM Revenue & Customs
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+package pages
+
+import controllers.routes
+import play.api.libs.json.JsPath
+import play.api.mvc.Call
+
+case object SavedProgressPage extends QuestionPage[String] {
+
+  override def path: JsPath = JsPath \ toString
+
+  override def toString: String = "continueUrl"
+
+  override def route(waypoints: Waypoints): Call =
+    routes.SavedProgressController.onPageLoad("continueUrl") // TODO: check logic
```

Comment:
SavedProgressPage in OSS doesn't have a route method https://github.com/hmrc/one-stop-shop-registration-frontend/blob/8d25bc6080d13845bcdc4f4b29db0116926b5874/app/pages/SavedProgressPage.scala